src/components/organisms/ChatSidebar/components/PrivateChats/components/index.ts

```diff
@@ -0,0 +1,3 @@
+export { OnlineUserList } from "./OnlineUserList";
+export { PrivateChatPreview } from "./PrivateChatPreview";
+export { RecipientChat } from "./RecipientChat";
```

Comment:
Yeah, good question. I have been thinking on the approach in such cases as well. The arguments for would be that it's not efficient to expose very specific components that are only used in one place into the `shared` folder. But eventually, we would probably come to a point, when such component is reused in a few places. When this situation happens, we could just extract it and place into `shared` folder

Reply:
My general approach in the past has been to have a single `FEATURE/components` folder, and anything 'private' to that feature is in there. (rather than multiple levels of nesting as we had here)